docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3"
```

Comment:
Well, I have mixed feelings about this, because we are instantiating 4 containers with the same images to run a different service each.

In general, an image should have only one service and lift that service, but I understand that this is the case because in our particular case we have to build all the rmf projects together with Colcon for it to work (AFAIK).

I'm thinking of this as a hybrid solution, not following the principle of separation of services in the image. But if we are following the theory of a service per container.

I think it worth leaving a comment on the reason of this architecture.

https://docs.docker.com/config/containers/multi-service_container/#:~:text=It's%20ok%20to%20have%20multiple,defined%20networks%20and%20shared%20volumes.